xbmc/interfaces/python/XBPython.cpp

```diff
@@ -581,6 +581,10 @@ bool XBPython::OnScriptInitialized(ILanguageInvoker *invoker)
       CLog::Log(LOGDEBUG, "PYTHONHOME -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
       CLog::Log(LOGDEBUG, "PYTHONPATH -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
     }
+
+#if defined(TARGET_DARWIN)
```

Comment:
imo this should be set before the application is launched, not in the code. it should not overwrite user set environment variables (overwrite flag -> 0)